### PR TITLE
Test rust graphql

### PR DESCRIPTION
**Pull request by Bernardstanislas**

### CODE REVIEW

File: .github/actions/backend-run/action.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      shell: bash
+      run: cp .env.example .env
+
+    - name: Build the app
+      shell: bash
+      run: cargo build
+
+    - name: Run the app
+      shell: bash
+      run: cargo run &> out.log &
```
Comment (lechinoix) [inline .github/actions/backend-run/action.yml:18]: 🧙‍♂️

File: docker-compose.yml [hunk 2]
```diff
@@ -55,6 +55,8 @@ services:
       BACKEND_GRAPHQL_URL: ${BACKEND_GRAPHQL_URL:-http://host.docker.internal:8000/graphql}
       HASURA_GRAPHQL_ENABLE_REMOTE_SCHEMA_PERMISSIONS: "true"
       HASURA_GRAPHQL_JWT_SECRET: ${HASURA_GRAPHQL_JWT_SECRET}
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```
Comment (lechinoix) [inline docker-compose.yml:59]: 🪄

**Review by lechinoix — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (Bernardstanislas) [reply docker-compose.yml:59]: https://stackoverflow.com/questions/31324981/how-to-access-host-port-from-docker-container

File: .github/actions/backend-run/action.yml (hunk 1 above)
Comment (Bernardstanislas) [reply .github/actions/backend-run/action.yml:18]: https://user-images.githubusercontent.com/4435377/203500961-472c2066-707b-4932-ab18-32ecfde6966f.mp4

**Pull request merged**